client/components/mma/switch/SwitchReview.tsx

```diff
@@ -127,8 +127,8 @@ export const SwitchReview = () => {
 
 	const confirmSwitch = async (amount: number) => {
 		try {
-			setIsSwitching(true);
-			const response = await productMoveFetch(false);
+			setIsSwitching(false);
```

Comment:
```suggestion
			setIsSwitching(true);
```
you switched the wrong one back :)